develop/learning-paths/themes/articles/01-setting-up-the-theme/04-modifying-theme-templates.markdown

```diff
@@ -0,0 +1,169 @@
+# Modifying Theme Templates
+
+In this section you'll add some finishing touches to `portal_normal.ftl` and
+`navigation.ftl` to match the finished mockup for the Lunar Resort.
+
+1. Open `portal_normal.ftl` and find the `<body class="${css_class}">` and 
+   replace it with the following code:
+   
+       <body class="${css_class} dockbar-split">
```

Comment:
Glad you're enjoying it. I've made quite a few changes since I last submitted a draft, after I received feedback from Robert. I'll double check this section of code to make sure I fixed it. Thanks for pointing it out.

-----Original Message-----
From: "Jim Hinkey" <notifications@github.com>
Sent: ‎8/‎22/‎2015 9:09 AM
To: "sez11a/liferay-docs" <liferay-docs@noreply.github.com>
Cc: "Michael Williams" <michael.williams@liferay.com>
Subject: Re: [liferay-docs] LRDOCS-1735 Created Themes learning path. (#1333)

In develop/learning-paths/themes/articles/01-setting-up-the-theme/04-modifying-theme-templates.markdown:
> @@ -0,0 +1,169 @@
> +# Modifying Theme Templates
> +
> +In this section you'll add some finishing touches to `portal_normal.ftl` and
> +`navigation.ftl` to match the finished mockup for the Lunar Resort.
> +
> +1. Open `portal_normal.ftl` and find the `<body class="${css_class}">` and 
> +   replace it with the following code:
> +   
> +       <body class="${css_class} dockbar-split">
This code isn't rendering; I think you need to indent it more.
I'm enjoying reading your learning path draft in preparation for sharing the Dev 1 slides on Themes.  
—
Reply to this email directly or view it on GitHub.